api/v1alpha1/servicebinding_types.go

```diff
@@ -53,10 +53,14 @@ type Environment struct {
 
 // These are valid conditions of ServiceBinding.
 const (
-	// ServiceBindingReady means the ServiceBinding has projected the secret
-	//  and the Workload is ready to start. It does not indicate the condition
-	// of either the Service or the Workload resources referenced.
-	ServiceBindingConditionReady = "Ready"
+	// ServiceBindingBoundCondition means the ServiceBinding has projected the secret
+	// into the Workload is and the binding is created.
+	// It does not indicate the condition of either the Service or the Workload resources referenced.
+	ServiceBindingBoundCondition = "Bound"
+	// ServiceBindingNotBoundCondition means the ServiceBinding has not projected the secret
+	// into the Workload.
+	// This will occur when the secret top be bound is not found
+	ServiceBindingNotBoundCondition = "NotBound"
```

Comment:
```suggestion
	// ServiceBindingBoundCondition means the ServiceBinding has successfully
	// projected the secret into the Workload.
	ServiceBindingBoundCondition = "Bound"
	// ServiceBindingNotBoundCondition means the ServiceBinding has not 
	// projected the secret into the Workload.
	// As an example, this will occur when the secret to be bound is not found
	ServiceBindingNotBoundCondition = "NotBound"
```